### PR TITLE
Fix path for security report line

### PR DESCRIPTION
**Pull request by inductor**
Fixes https://github.com/opencontainers/runc/issues/2220

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Currently, the following features are not considered to be production-ready:
 
 ## Security
 
-The reporting process and disclosure communications are outlined in [/org/security](https://github.com/opencontainers/org/blob/master/security/).
+The reporting process and disclosure communications are outlined in [/org/SECURITY.md](https://github.com/opencontainers/org/blob/master/SECURITY.md).
```
Comment (cyphar) [inline README.md]: Replace `/org/SECURITY.md` with `here`.

File: SECURITY.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
 # Security
 
-The reporting process and disclosure communications are outlined in [/org/security](https://github.com/opencontainers/org/blob/master/security/).
+The reporting process and disclosure communications are outlined in [/org/SECURITY.md](https://github.com/opencontainers/org/blob/master/SECURITY.md).
```
Comment (cyphar) [inline SECURITY.md]: Same comment here.

Comment (inductor) [reply SECURITY.md]: @cyphar Fixed!

**Comment by cyphar** [comment]
LGTM.

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**